src/components/NewMovie/NewMovie.tsx

```diff
@@ -34,7 +34,8 @@ export const NewMovie: React.FC<Props> = ({ onAdd }) => {
     setImdbId('');
   };
 
-  const isButton = !title || !description || !imgUrl || !imdbUrl || !imdbId;
+  const isButtonAvailable = !title.trim() || !description.trim()
```

Comment:
```suggestion
  const isButtonAvailable = !title.trim()
```

The description can be empty